tools/UpdatePaths/Scripts/78733_simple_to_basic_raw_prophet.txt

```diff
@@ -0,0 +1 @@
+/mob/living/simple_animal/hostile/heretic_summon/raw_prophet : /mob/living/basic/heretic_summon/raw_prophet
```

Comment:
```suggestion
/mob/living/simple_animal/hostile/heretic_summon/raw_prophet : /mob/living/basic/heretic_summon/raw_prophet{@OLD;AIStatus=@SKIP;stop_automated_movement=@SKIP}
```

lets be nicer to downstreams and the such in case they're cooking